docs/source/example/dscp-app.md

```diff
@@ -0,0 +1,45 @@
+
+# DSCP - Digital Supply Chain Platform
+
+### A sample supply chain application deployed on Substrate based DLT network using Hyperledger Bevel <br>
+<br>
+This demo application has been developed to demonstrate the benefits of distributed systems to the additive-manufacturing industry within aerospace. The application defines a consortium of multiple organizations. It allows nodes to track products or goods along with their chain of custody. It provides the members of the consortium all the relevant data about their product throughout it's journey, from raw material to the finished product. It provides simple web clients that let you perform the actions of different personas in a supplychain industry.<br>
+<br>
+
+### Pre-requisites
+
+***
+
+- Substrate network of one or more organizations. A complete supplychain network will have the following organizations:
+  - Supplychain (admin organization)
```

Comment:
Is this the correct name for the owner org?

Reply:
These are generic names for a common supply chain network. 'oem', tierone' etc are specific to our project and not very intuitive for the readers/users of this repo.

Reply:
While I agree with this, the majority of users that are deploying this example app would use the sample network provided. Maybe specify these are example names, or we should update our sample network to use these generic names.